doc/classes/Node3D.xml

```diff
@@ -49,7 +49,8 @@
 		<method name="get_parent_node_3d" qualifiers="const">
 			<return type="Node3D" />
 			<description>
-				Returns the parent [Node3D], or an empty [Object] if no parent exists or parent is not of type [Node3D].
+				Returns the parent [Node3D], or an empty [Object] if no parent exists, the parent is not of type [Node3D], or [member top_level] is [code]true[/code].
```

Comment:
```suggestion
				Returns the parent [Node3D], or [code]null[/code] if no parent exists, the parent is not of type [Node3D], or [member top_level] is [code]true[/code].
```

Reply:
Could we get team consensus before I change this again? I had it as null originally but was told to change it to empty object.

Reply:
My bad didn't see that, where was that discussed? If not `null` then I'd say it should be:
```suggestion
				Returns the parent [Node3D], or an empty [Node3D] if no parent exists, the parent is not of type [Node3D], or [member top_level] is [code]true[/code].
```

Reply:
It was in the contributors chat, so it actually was included in my initial commit. Sorry I misremembered it as being a discussion that happened here. I asked about it because the rest of the docs say "empty [Object]", so I was asking about the distinction. The rationale given by a team member was that it's technically different from a NIL variant. I'm also of the opinion that null would be more informative, but yeah I'd just like to avoid changing it back again haha. Sorry hope you understand.

Reply:
That's fair :) Just wasn't aware of a discussion going on elsewhere (despite being active in the docs chat), worth fixing later as it doesn't really return an empty `Object`

Reply:
I did say that there is a distinction, but in practice it does not matter to the outsider. `Object<null> == null` in GDScript.

Reply:
You're right, there are many cases of `empty [...]`, but the instance that I'm replacing here seems to be _the_ only instance of `empty [Object]`! 😅 I will change it to `null`. Am I correct in assuming that all the other instances of `empty [...]` should eventually be replaced with `null` as well for consistency and clarity?

Reply:
Not generally no, for simple types it doesn't apply, `StringName` can't be `null`

Reply:
Gotcha, so only for `RefCounted` then?